tests/files/tests_multiple_sudoers_pingers.ok

```diff
@@ -0,0 +1,11 @@
+#
+# Ansible managed
+#
+# system_role:sudo
+
+
```

Comment:
I should fix the template to avoid creating extra blank lines. Let's keep the test .ok configs in a proper format and then fix template to remove extra newlines.

Reply:
Done